### ISSUE TITLE
AttributeError "can't set attribute" when trying to display a polymorphic form

### ISSUE THREAD

**Issue by TiphaineLAURENT** [django-polymorphic/django-polymorphic#391]
The traceback :
```py
Environment:


Request Method: GET
Request URL: https://<domain>/admin/tutorials/tutorial/add/

Django Version: 2.2.1
Python Version: 3.5.3
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'debug_toolbar',
 'tiphainelaurent.staticfiles.MyStaticFilesConfig',
 'rest_framework',
 'rest_framework.authtoken',
 'nested_admin',
 'polymorphic',
 'django_user_agents',
 'api',
 'showcase',
 'tutorials',
 'games',
 'cvs']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware',
 'django_user_agents.middleware.UserAgentMiddleware']



Traceback:

File "/home/web/venv/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/home/web/venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/home/web/venv/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/contrib/admin/options.py" in wrapper
  606.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/contrib/admin/options.py" in add_view
  1634.         return self.changeform_view(request, None, form_url, extra_context)

File "/home/web/venv/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  45.         return bound_method(*args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "/home/web/venv/lib/python3.5/site-packages/django/contrib/admin/options.py" in changeform_view
  1522.             return self._changeform_view(request, object_id, form_url, extra_context)

File "/home/web/venv/lib/python3.5/site-packages/django/contrib/admin/options.py" in _changeform_view
  1596.             media = media + inline_formset.media

File "/home/web/venv/lib/python3.5/site-packages/nested_admin/nested.py" in media
  109.                 media = media + inline.media

File "/home/web/venv/lib/python3.5/site-packages/nested_admin/nested.py" in media
  105.         media = media + self.formset.media

File "/home/web/venv/lib/python3.5/site-packages/polymorphic/formsets/models.py" in media
  236.             add_media(media, form.media)

File "/home/web/venv/lib/python3.5/site-packages/polymorphic/formsets/utils.py" in add_media
  15.         dest._css = combined._css

Exception Type: AttributeError at /admin/tutorials/tutorial/add/
Exception Value: can't set attribute
```

**Comment by TiphaineLAURENT** [comment]
I posted the same issue on the django-nested-admin github because I don't exactly which project has the bug.

**Comment by AlirezaJalali** [comment]
No way around it till it gets fixed?

**Comment by TiphaineLAURENT** [comment]
I didn't find anything yet. Since it is only an admin interface issue, it is not critical on my side.

**Comment by AlirezaJalali** [comment]
Since I need the admin interface to work properly, I may have to change my models not to be polymorphic anymore, which is very sad.

**Comment by wfehr** [comment]
Seems like a duplicate of #382.

My "workaround" was to use Django 2.1.

**Comment by vdboor** [comment]
Fixed in 622e1e780b84a8e04c5af2d6758fb457ff92ea93!
